tox.ini

```diff
@@ -125,6 +125,7 @@ envlist =
 # but these tests still work.
```

Comment:
We also need to add the tests to `.circleci/config.yml` to get the tests to run in CI. You can use `redis` or `boto` as an example 🙂